index/server/vendor/github.com/devfile/registry-support/index/generator/schema/schema.go

```diff
@@ -59,5 +59,6 @@ type Schema struct {
 	ProjectType     string            `yaml:"projectType,omitempty" json:"projectType,omitempty"`
 	Language        string            `yaml:"language,omitempty" json:"language,omitempty"`
 	Links           map[string]string `yaml:"links,omitempty" json:"links,omitempty"`
+	Resources       []string          `yaml:"resources,omitempty" json:"resources,omitempty"`
```

Comment:
Can we add the definition and example for the new field `Resources`?

Reply:
This is just the vendored changes being pulled in from the generator, it needs to be updated there.

Reply:
Since the generator and index server live in the same repo, I added a commit to include examples in the index schema.

Reply:
I see 👍